### ISSUE TITLE
Update link to Fabrik documentation on homepage

### ISSUE THREAD

**Issue by nikochiko** [Cloud-CV/CloudCV#114]
- Currently, the link to the Fabrik documentation in the "PROJECTS" section of the [homepage](https://cloudcv.org/) points to the outdated [Fabrik Wiki](https://github.com/Cloud-CV/Fabrik/wiki) (see this Fabrik issue: https://github.com/Cloud-CV/Fabrik/issues/522).
- For consistency, because the documentations of both EvalAI and Origami point to their readthedocs.io pages, it would be better to change this link to https://fabrik.readthedocs.io/en/latest/ .

**Comment by sohamsshah** [comment]
Hey there! If this issue is still open to solve, I would love to solve this!

**Comment by sohamsshah** [comment]
Hey @nikochiko Can I work on this?

**Comment by nikochiko** [comment]
@sohamsshah Sure! Go ahead.

**Comment by sohamsshah** [comment]
Hey, I went through the codebase and I suppose that the url that links to the fabrik project is actually in that DB, which is hosted on AWS RDB. Can you assist me upon that? Or if any better/effective alternative is available, I would be ready to implement it.

**Comment by nikochiko** [comment]
Yeah I think you're right. I suggest you take on some other issues for now, maybe in [Cloud-CV/EvalAI](https://github.com/Cloud-CV/EvalAI). 
Docs link isn't such a big deal, but I'll just ping @RishabhJain2018 to let him know.

(Link to mentioned code - [source](https://github.com/Cloud-CV/CloudCV/blob/develop/frontend/src/components/home/homeBody/HomeShowcase.js#L19-L28))
